extension-readme.md

```diff
@@ -82,53 +87,66 @@ For complete setup instructions for these three components, see [Installing the
 
 # <a id="features"></a>Features
 
--   [Serverless Applications, Lambda Functions, and CloudFormation Stacks](#sam-and-lambda)
--   [AWS CDK Applications](#cdk-apps)
--   [Amazon EventBridge Schemas](#eventbridge)
--   [Amazon ECS Task Definition Files](#ecs-files)
+-   [AWS Serverless Applications](#sam-and-lambda)
+-   [AWS CDK applications](#cdk-apps)
+-   [Amazon EventBridge schemas](#eventbridge)
+-   [Amazon ECS task definition files](#ecs-files)
 -   [AWS Step Functions](#sfn-files)
 
 { [Return to Contents](#contents) } or { [Return to Top](#top) }
 
 ---
 
-## <a id="sam-and-lambda"></a>Serverless Applications, Lambda Functions, and CloudFormation Stacks
+## <a id="sam-and-lambda"></a> AWS Serverless Applications 
+
+The AWS Toolkit enables you to develop [AWS serverless applications](https://aws.amazon.com/serverless/) locally. It also allows you to use _CodeLens_&mdash;a Visual Studio Code feature&mdash;to do the following:
```

Comment:
I don't think you need to explain what a CodeLens is, this will appear on the VS Code marketplace or as a welcome page on installing into the toolkit.